src/overlays/actors/ovl_En_Zl4/z_en_zl4.c

```diff
@@ -180,24 +180,24 @@ static struct_80034EC0_Entry sAnimationEntries[] = {
 
 #include "z_en_zl4_cutscene_data.c"
 
-void EnZl4_SetCsCameraAngle(GlobalContext* globalCtx, s16 index) {
+void EnZl4_SetActiveCamDir(GlobalContext* globalCtx, s16 index) {
     Camera* activeCam = GET_ACTIVE_CAM(globalCtx);
 
     Camera_ChangeSetting(activeCam, CAM_SET_FREE0);
-    activeCam->at = sCsCameraAngle[index].at;
-    activeCam->eye = activeCam->eyeNext = sCsCameraAngle[index].eye;
-    activeCam->roll = sCsCameraAngle[index].roll;
-    activeCam->fov = sCsCameraAngle[index].fov;
+    activeCam->at = sActiveCamDir[index].at;
```

Comment:
I don't think `sActiveCamDir`/`sActiveCamMove` should mention the target camera being the active one, since it's not relevant to the data
Until a further documentation pass maybe gives it an actual name maybe just `sCamDirections`